src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets

```diff
@@ -261,12 +261,11 @@
 
         <!-- Invoke the GetMauiItems target on all project references -->
         <!-- This will accumulate images into our MauiImage group -->
-        <!--<MSBuild Targets="GetMauiItems" Projects="@(_MSBuildProjectReferenceExistent)">-->
         <MSBuild
             Targets="GetMauiItems"
-            Projects="@(ProjectReference)"
+            Projects="%(_ResolvedProjectReferencePaths.OriginalProjectReferenceItemSpec)"
             BuildInParallel="true"
-            Properties="TargetFramework=$(TargetFramework)"
+            Properties="TargetFramework=%(_ResolvedProjectReferencePaths.NearestTargetFramework)"
```

Comment:
One difference here is that `%` will "batch" the call running `<MSBuild/>` N times. Where before it would run it a single time passing in multiple projects.

Is that a problem?

Reply:
For the first question, you just discovered a huge bug. Before, it would run a single msbuild with a single TFM, but the MSBuild task does not actually support that. What happens is that there is a build for all the projects, but just the last one to complete actually sets the output items. 

So multiple builds is actually the correct way.

Reply:
For the second question, I will first filter out the items that are missing these values to be safe, but they should always have them.

Reply:
I think if `%(OriginalProjectReferenceItemSpec)` is blank it should just use `%(Identity)` and `%(NearestTargetFramework)` fall back to `$(TargetFramework)`?

MSBuild is so loosely typed these could randomly become blank in the future.

Reply:
Hmm, Identity is the path of the dll since the `_ResolvedProjectReferencePaths` item group is the result of the `ResolveProjectReferences` target.